internal/cmd/rename.go

```diff
@@ -0,0 +1,61 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2019 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/eclipse-iofog/iofogctl/internal/rename"
+	"github.com/eclipse-iofog/iofogctl/pkg/util"
+	"github.com/spf13/cobra"
+)
+
+func newRenameCommand() *cobra.Command {
+	// Instantiate command
+	cmd := &cobra.Command{
```

Comment:
Should fix the number of expected arguments, and specify the list of valids arguments